src/content/app/genome-browser/hooks/useGenomeBrowserAnalytics.ts

```diff
@@ -0,0 +1,307 @@
+    selectedCog: string,
+    shouldApplyToAll: boolean
+  ) => {
+    sendTrackEvent({
+      category: 'track_settings',
+      label: selectedCog,
+      action: 'apply_to_all - ' + (shouldApplyToAll ? 'unselected' : 'selected')
+    });
+  };
+
+  const trackTrackPanelTabChange = (selectedTrackPanelTab: TrackSet) => {
+    sendTrackEvent({
+      category: 'track_panel_tab',
+      label: selectedTrackPanelTab,
+      action: 'selected'
+    });
+  };
+
+  const trackTrackPanelSectionToggled = (sectionName: string) => {
+    sendTrackEvent({
+      category: 'track_panel',
+      label: sectionName,
+      action: 'section_toggled'
+    });
+  };
+
+  const trackFocusObjectReset = () => {
+    sendTrackEvent({
+      category: 'reset_icon',
+      action: 'clicked'
+    });
+  };
+
+  const trackSidebarModalViewToggle = (view: string) => {
+    sendTrackEvent({
+      category: 'sidebar_toolstrip',
+      action: 'modal_opened',
+      label: view
+    });
+  };
+
+  const trackDrawerOpened = (drawerView: string) => {
+    sendTrackEvent({
+      category: 'track_drawer',
+      action: 'opened',
+      label: drawerView
+    });
+  };
+
+  const trackDrawerSequenceViewed = (sequenceType: string) => {
+    sendTrackEvent({
+      category: 'track_drawer',
+      action: 'sequence_viewed',
+      label: sequenceType
+    });
+  };
+
+  const trackDrawerSequenceCopied = (sequenceType: string) => {
+    sendTrackEvent({
+      category: 'track_drawer',
+      action: 'sequence_copied',
+      label: sequenceType
+    });
+  };
+
+  const trackDrawerSequenceDownloaded = (selectedOptions: string) => {
+    sendTrackEvent({
+      category: 'track_drawer',
+      action: 'sequence_downloaded',
+      label: selectedOptions
+    });
+  };
+
+  const trackSpeciesChangeLinkClicked = () => {
+    sendTrackEvent({
+      category: 'app_bar',
+      action: 'change_link_clicked'
+    });
+  };
+
+  const trackGenomeChanged = () => {
+    sendTrackEvent({
+      category: 'app_bar',
+      action: 'change_genome'
+    });
+  };
+
+  const trackTrackPanelToggled = (isOpened: boolean) => {
+    sendTrackEvent({
+      category: 'track_panel',
+      action: isOpened ? 'opened' : 'closed'
+    });
+  };
+
+  const trackTrackVisibilityToggled = (trackName: string, status: boolean) => {
+    sendTrackEvent({
+      category: 'regular_track_visibility',
+      action: 'turned_' + status ? 'on' : 'off',
+      label: trackName
+    });
+  };
+
+  const trackFocusTrackVisibilityToggled = (action: Status) => {
+    sendTrackEvent({
+      category: 'focus_track_visibility',
+      action: action === Status.SELECTED ? 'turned_on' : 'turned_off'
+    });
+  };
+
+  const trackTranscriptInTrackVisibilityToggled = (
+    transcriptStableId: string,
+    status: boolean
+  ) => {
+    sendTrackEvent({
+      category: 'transcript_in_track_visibility',
+      action: 'turned_' + status ? 'on' : 'off',
+      label: `transcript-${transcriptStableId}`
+    });
+  };
```

Comment:
Is this correct payload? Event: "on"? Even when a transcript is hidden? I am not sure what I am looking at:

https://user-images.githubusercontent.com/6834224/189762397-a82efdf4-e0d8-422b-bcf2-617e2b6b5683.mov

Reply:
This is now fixed